lib/recode/issue.ex

```diff
@@ -1,24 +0,0 @@
-defmodule Recode.Issue do
```

Comment:
Just a minor nit/idea, but what do you think of keeping `Issue` in Recode instead of moving it to Rewrite? 

Seems to me that Rewrite shouldn't be responsible for defining what an issue is - because it is _API for rewriting sources_. D

Reply:
@NickNeck seems like a good plan! ✔️